activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeClaimReleaseTest.java

```diff
@@ -0,0 +1,149 @@
+package org.activiti.spring.boot.tasks;
+
+import org.activiti.runtime.api.NotFoundException;
+import org.activiti.runtime.api.TaskAdminRuntime;
+import org.activiti.runtime.api.TaskRuntime;
+import org.activiti.runtime.api.model.Task;
+import org.activiti.runtime.api.model.builders.TaskPayloadBuilder;
+import org.activiti.runtime.api.query.Page;
+import org.activiti.runtime.api.query.Pageable;
+import org.activiti.runtime.api.security.SecurityManager;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.test.context.support.WithUserDetails;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@ContextConfiguration
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
I don't like the idea of establishing an execution order for tests and having them depending on each other. This kind of test becomes a nightmare to maintain. I don't have a solution for now on how to solve the "switch of user" problem, but I'd really like to avoid adding this.